source/_integrations/plugwise.markdown

```diff
@@ -57,6 +57,18 @@ For example, if you have an Adam setup with a Lisa named 'Living' and a Tom name
 
 Centralized measurements such as 'power' for a P1, 'outdoor_temperature' on Anna or Adam will be assigned to your gateway entity. Heating/cooling device measurements such as 'boiler_temperature' will be assigned to the OpenTherm/OnOff entity.
 
+## Data updates
+
+The interval which the integration fetches data from the Smile depends on the device:
+
+- Power entities, such as the P1, will be refreshed every 10 seconds.
+- Climate entities will be refreshed every 60 seconds.
+- Stretch entities will be refreshed every 60 seconds.
+
+## Removing the integration
+
+This integration follows standard integration removal. No extra steps are required within Home Assistant or on your Plugwise devices.
```

Comment:
Please include the snippet with the standard removal instructions (similar to the include econfig flow includes)

Reply:
Tnx for that - in the same light the configuration section indeed would also benefit from the includes - though to keep things small, will do that in a follow up PR.

Reply:
Reference #36087